RFS/src/main/java/org/opensearch/migrations/bulkload/worker/IndexRunner.java

```diff
@@ -35,16 +36,34 @@ public IndexMetadataResults migrateIndices(MigrationMode mode, ICreateIndexConte
         };
         var results = IndexMetadataResults.builder();
 
+        // log out filtered items
+        repoDataProvider.getIndicesInSnapshot(snapshotName)
+                .stream()
+                .filter(Predicate.not(FilterScheme.filterIndicesByAllowList(indexAllowlist, logger)))
+                .forEach(index -> {
+                    var indexMetadata = metadataFactory.fromRepo(snapshotName, index.getName());
+                    log.atInfo().setMessage("{ \"before\": {},\n\"after\":{}}")
+                            .addArgument(indexMetadata)
+                            .addArgument("Removed due to index filter")
+                            .log();
```

Comment:
Add a CreationResult for these items with a new failure type so the UI can display these results.

Reply:
Glad you reviewed @peternied; wouldn't have caught that one!